### PR TITLE
docs: add documentation to explain the extension demo

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: README.md
Comment (ergunsh) [inline README.md:227]:
```suggestion
You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extend DevTools.
```

File: README.md
Comment (ergunsh) [inline README.md:231]:
```suggestion
To load the example into Chrome DevTools. Follow these steps:
```

File: README.md
Comment (ergunsh) [inline README.md:236]: `A new **Export as a Custom JSON script** menu is added` sentence feels a bit out of the flow. Maybe we can say `Now you can see a new **Export as a Custom JSON script** action in the export menu`. WDYT?

File: README.md
Comment (ergunsh) [inline README.md:240]: nit: as far as I understand, this `[]` part is added as an `alt` tag to the image. So, we can try to be descriptive there instead of action oriented. We can say something like `Demo video that shows how to extend export options in Recorder panel by adding a Chrome extension`. WDYT?

**Review by ergunsh — COMMENTED** [review]
LGTM with a few typo fixes and a few nits.

I think we might want to wait until M104 stable until merge this change. WDYT?

**Comment by OrKoN** [comment]
I think we don't need to merge but let's mention that extension API is available in M104+.

File: README.md
Comment (jecfish) [inline README.md:236]:
```suggestion
4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). Now you can see a new **Export as a Custom JSON script** action in the export menu.
```

File: README.md
Comment (jecfish) [inline README.md:240]:
```suggestion
[![Demo video that shows how to extend export options in Recorder panel by adding a Chrome extension](https://user-images.githubusercontent.com/5917927/172872574-15ad8bea-142a-4972-bf1d-bf1379a955ba.png)](https://youtu.be/TCxIfbxgypQ)
```

File: README.md
Comment (jecfish) [inline README.md:236]:
```suggestion
4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). Now you can see a new **Export as a Custom JSON script** option in the export menu.
```

File: README.md
Comment (jecfish) [inline README.md:226]:
```suggestion
You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extend DevTools.

This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to test the latest features in Chrome.
```

File: README.md
Comment (jecfish) [inline README.md:228]:
```suggestion
This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to try out the latest bleeding edge features in Chrome.
```

File: README.md
Comment (jecfish) [inline README.md:228]:
```suggestion
This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to try out cutting-edge features in Chrome.
```

**Comment by jecfish** [comment]
I think we can merge when it's approved. I've added mention that it is available from M104+.

File: README.md
Comment (OrKoN) [inline README.md:216]: we don't need this (about CJS) anymore as it was outdated

File: README.md
Comment (ergunsh) [inline README.md:228]:
```suggestion
This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider installing [Chrome Canary](https://www.google.com/chrome/canary/) to try out cutting-edge features in Chrome.
```

**Review by ergunsh — APPROVED** [review]
LGTM with final nit/typo 😹

File: README.md
Comment (OrKoN) [inline README.md:223]:
```suggestion
```

**Pull request merged**
